test/unit/presenters/formats/answer_presenter_test.rb

```diff
@@ -0,0 +1,122 @@
+        body: 'some content'
+                                   )
+
+      @presenter = Formats::AnswerPresenter.new(@edition)
+
+      @expected_attributes_for_publishing_api_hash = {
+        title: @edition.title,
+        base_path: "/#{@edition.slug}",
+        description: "",
+        schema_name: "answer",
+        document_type: artefact.kind,
+        need_ids: [],
+        public_updated_at: @edition.public_updated_at,
+        publishing_app: "publisher",
+        rendering_app: "frontend",
+        routes: [
+          { path: "/#{@edition.slug}", type: "exact" },
+          { path: "/#{@edition.slug}.json", type: "exact" }
+        ],
+        redirects: [],
+        update_type: "major",
+        change_note: @edition.change_note,
+        details: {
+          body: [
+            {
+              content_type: 'text/govspeak',
+              content: 'some content'
+            }
+          ],
+          external_related_links: expected_external_related_links,
+        },
+        locale: 'en',
+      }
+    end
+
+    should "create an attributes hash for the publishing api" do
+      assert_equal @expected_attributes_for_publishing_api_hash, @presenter.render_for_publishing_api(republish: false)
+    end
+
+    should "create an attributes hash for the publishing api for a republish" do
+      attributes_for_republish = @expected_attributes_for_publishing_api_hash.merge(update_type: "republish",)
+      presented_hash = @presenter.render_for_publishing_api(republish: true)
+      assert_equal attributes_for_republish, presented_hash
+      assert_valid_against_schema(presented_hash, 'answer')
+    end
+
+    should 'create an attributes hash for a minor change' do
+      @edition.update_attribute(:major_change, false)
+
+      output = @presenter.render_for_publishing_api(republish: false)
+      assert_equal 'minor', output[:update_type]
+    end
+
+    should 'always return a "major" update_type for a first edition' do
+      first_edition = FactoryGirl.create(:edition, major_change: false, version_number: 1)
+      presenter = Formats::AnswerPresenter.new(first_edition)
+
+      output = presenter.render_for_publishing_api(republish: false)
+      assert_equal 'major', output[:update_type]
+    end
+  end
+
+  context ".render_for_publishing_api with a draft document" do
+    setup do
+      artefact = FactoryGirl.create(:artefact,
+        content_id: SecureRandom.uuid,
+        language: 'cy',
+                                   )
+      updated_at = 1.minute.ago
+      @edition = FactoryGirl.create(
+        :answer_edition,
+        state: "draft",
+        updated_at: updated_at,
+        panopticon_id: artefact.id,
+      )
+      @output = Formats::AnswerPresenter.new(@edition).render_for_publishing_api
+    end
+
+    should "be valid against schema" do
+      assert_valid_against_schema(@output, 'answer')
+    end
+
+    should 'use updated_at value if public_updated_at is nil' do
```

Comment:
There's a couple of tests here that we might not need for the specialised/inheriting presenters... `public_updated_at` and `locale` could just be tested against the EditionFormatPresenter?